homeassistant/components/rituals_perfume_genie/binary_sensor.py

```diff
@@ -9,9 +9,9 @@
     DEVICE_CLASS_BATTERY_CHARGING,
     BinarySensorEntity,
 )
+from homeassistant.components.rituals_perfume_genie import RitualsDataUpdateCoordinator
```

Comment:
Use relative imports within the same module

```suggestion
from . import RitualsDataUpdateCoordinator
```